report_templates/entitlements.erb

```diff
@@ -0,0 +1,41 @@
+<%#
+name: Entitlements
+snippet: false
+model: ReportTemplate
+require:
+- plugin: katello
+  version: 3.14.0
+%>
+
+<%- load_hosts(includes: [:lifecycle_environment, :operatingsystem, :architecture, :content_view, :organization, :reported_data, :subscription_facet, :pools => [:subscription]]).each_record do |host| -%>
+  <%- host.pools.each do |pool| -%>
```

Comment:
usually we do the indentation inside of erb tag, see https://github.com/theforeman/community-templates/blob/develop/report_templates/applicable_errata.erb#L24-L26 for example